### DISCUSSION TITLE
Where do we have to submit the project proposal/assignment?

### DISCUSSION THREAD

**Discussion by Himabindugssn** [github/india#669] [Students]
For Symbl.ai, it's written: Complete the [Assignment] and submit it on our portal with proper implementation and screenshots in PDF Format.

But there's no link given of any portal :/

https://externship.github.in/organization/symbl.ai/61aa3412e5b069a2dc6b544c

**Comment by vikramadityasinghs** [comment]
I have the same doubt could anyone clarify?

**Comment by Rupam-Shil** [comment]
Got any answer? I have the same doubt
